airflow/ui/src/pages/DagsList/Dag/Graph/TaskNode.tsx

```diff
@@ -63,7 +65,7 @@ export const TaskNode = ({
           <Box>
             {/* TODO: replace 'Operator' */}
             <Text fontSize="xs" fontWeight="lighter" textTransform="uppercase">
-              {isGroup ? "Task Group" : "Operator"}
+              {isGroup ? "Task Group" : nodeType}
```

Comment:
```suggestion
              {isGroup ? "Task Group" : operator}
```

Actually the idea was to put the operator here.

Reply:
Updated, screenshot updated too.

I have added the `nodeType` bellow as a simple text, it's just so that we have it available in the component, later we can replace the text by nice icons or anything else to differentiate asset, tasks, etc...